site/content/en/v0.6.0/install/install-egctl.md

```diff
@@ -52,6 +52,6 @@ curl -fsSL https://gateway.envoyproxy.io/get-egctl.sh | VERSION=latest bash
 
 {{% alert title="Next Steps" color="warning" %}}
 
-You can refer to [User Guides](/latest/user/egctl) to more details about egctl.
+You can refer to [User Guides](/latest/user/operations/egctl) to more details about egctl.
```

Comment:
```suggestion
You can refer to [User Guides](../user/operations/egctl) to more details about egctl.
```

Reply:
It does not exist in the v.0.6.0 doc. Do we need to link it to the latest url or `../user/egctl` ? i personally prefer the latter.